### PR TITLE
bugzilla: be more explicit as to how to move forward

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: prow/plugins/bugzilla/bugzilla.go [hunk 1]
```diff
@@ -1006,7 +1006,7 @@ func handleMerge(e event, gc githubClient, bc bugzilla.Client, options plugins.B
 	unmergedMessage := fmt.Sprintf(`The following pull requests linked via external trackers have not merged:
 %s
 
-These pull request must merge or be unlinked from the Bugzilla bug in order for it to move to the next state.
+These pull request must merge or be unlinked from the Bugzilla bug in order for it to move to the next state. Once unlinked, request a bug refresh with <code>/bugzilla refresh</code>.
```
Comment (Miciah) [inline prow/plugins/bugzilla/bugzilla.go:1009]: "These pull request" → "These pull requests" (or "This pull request").  But that typo was already there.

**Comment by Miciah** [comment]
Thanks!  There is a typo in the message that you are modifying, but it was already present.  
/lgtm

File: prow/plugins/bugzilla/bugzilla.go (hunk 1 above)
Comment (petr-muller) [reply prow/plugins/bugzilla/bugzilla.go:1009]: Looks like there's a trailing whitespace that makes tests fail.

**Comment by stevekuznetsov** [comment]
/retest

I don't understand that failure whatsoever

**Review by alvaroaleman — APPROVED** [review]

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**